### PR TITLE
Improve bundles

### PR DESCRIPTION
**Pull request by Marinofull**
Now the developer just need to initialize the submodules with `git submodule update --init --recursive` or run the `install.sh`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # My vim files
 
-You just need to download `install.sh` script and give proper permissions `chmod u+x install.sh`. Then, run then script `./install.sh` and be happy!
+### Install
+    git clone https://github.com/cristianossd/myvimfiles ~/.vim
+    cd ~/.vim
+    git submodule update --init --recursive
+Now enjoy it!
+<!-- You just need to download `install.sh` script and give proper permissions `chmod u+x install.sh`. Then, run then script `./install.sh` and be happy! -->
```
Comment (cristianossd) [inline README.md]: You should remove this comment. It'll be unnecessary.

**Comment by cristianossd** [comment]
After you remove the comment block, group your commits into just one.

**Comment by Marinofull** [comment]
working on it

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[submodule "bundle/ctrlp.vim"]
+	path = bundle/ctrlp.vim
+	url = https://github.com/ctrlpvim/ctrlp.vim
```
Comment (Marinofull) [inline .gitmodules:3]: @cristianossd the kien repo sugest to use this fork

Comment (cristianossd) [inline .gitmodules:3]: Ok!

**Comment by Marinofull** [comment]
:smiley:

**Pull request merged**